addon/components/flexberry-lookup.js

```diff
@@ -692,12 +714,16 @@ export default FlexberryBaseComponent.extend({
     this.get('lookupEventsService').on('lookupDialogOnVisible', this, this._setModalIsVisible);
     this.get('lookupEventsService').on('lookupDialogOnHidden', this, this._setModalIsHidden);
 
-    // TODO: This is necessary because of incomprehensible one-way binding on new detail form, perhaps the truth is out there, but I did not find it.
-    this.addObserver('value', this, this._valueObserver);
-    this.addObserver('displayAttributeName', this, this._valueObserver);
-    this.addObserver(`relatedModel.${this.get('relationName')}`, this, this._valueObserver);
+    if (this.get('autocompletePersistValue') && Ember.isNone(this.get('value'))) {
+      this.set('autocompletePersistValueCache', this.get('displayValue'));
+    }
   },
 
+  /**
+    Observe lookup value changes.
+  */
+  valueObserver: Ember.on('init', Ember.observer('value', 'displayAttributeName', function() { Ember.run.once(this, '_valueObserver'); })),
```

Comment:
I think that if you remove the `Ember.on('init', ...`, then nothing will change.